lib/inspector/promises.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const inspector = require('inspector');
+const { promisify } = require('util');
+class Session extends inspector.Session {
+  #post = promisify(super.post.bind(this));
+  /**
+   * Posts a message to the inspector back-end.
+   * @param {string} method
+   * @param {Record<unknown, unknown>} [params]
+   * @returns {Promise}
```

Comment:
Is there any specific type for the Promise response? Is the type somehow related to the `params` param? (Can we maybe give a different name to the param? As a param called `params` could lead to some confusion?